src/app/featureFlags/constants.ts

```diff
@@ -0,0 +1,14 @@
+import { FeatureFlagVariantValue } from './types';
+
+export const experiments: { [key: string]: FeatureFlagVariantValue[] } = {
```

Comment:
i think typescript's [record](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype) type is supposed to be used for situations like this